code/__HELPERS/lighting.dm

```diff
@@ -1,9 +1,29 @@
 /// Produces a mutable appearance glued to the [EMISSIVE_PLANE] dyed to be the [EMISSIVE_COLOR].
 /proc/emissive_appearance(icon, icon_state = "", atom/offset_spokesman, layer = FLOAT_LAYER, alpha = 255, appearance_flags = NONE, offset_const)
```

Comment:
you still have the alpha argument

Reply:
part of me thinks it could still be implemented, and I don't have the guts to convince myself otherwise